docs/package/gluon-radv-filterd.rst

```diff
@@ -13,29 +13,31 @@ Selected router
 The router selection mechanism is independent from the batman-adv gateway mode.
 In contrast, the device originating the router advertisement could be any router
 or client connected to the mesh, as radv-filterd captures all router
-advertisements originating  from it. All nodes announcing router advertisement
+advertisements originating from it. All nodes announcing router advertisement
 **with** a default lifetime greater than 0 are being considered as candidates.
 
 In case a router is not a batman-adv originator itself, its TQ is defined by
 the originator it is connected to. This lookup uses the batman-adv global
 translation table.
 
-Initially the router is the selected by choosing the candidate with the
+Initially the router is the "selected" by choosing the candidate with the
 strongest TQ. When another candidate can provide a better TQ metric it is not
-picked up as the selected router until it will outperform the currently
-selected router by X metric units. The hysteresis threshold is configurable
-and prevents excessive flapping of the gateway.
+picked up as the "selected" router until it will outperform the currently
```

Comment:
why adding the quotes?
also: "until it outperforms the currently selected"

Reply:
because "selected" is a new naming convention, defined here for the paragraph.

We could also say "the router that is selected" instead